pulp_rpm/tests/functional/api/test_pulpimport.py

```diff
@@ -4,326 +4,227 @@
-            remote = cls.remote_api.create(body)
-
-            repository_sync_data = RpmRepositorySyncURL(remote=remote.pulp_href)
-            sync_response = cls.repo_api.sync(export_repo.pulp_href, repository_sync_data)
-            monitor_task(sync_response.task)
-            # remember it
-            export_repos.append(export_repo)
-            import_repos.append(import_repo)
-            remotes.append(remote)
-        return import_repos, export_repos, remotes
-
-    @classmethod
-    def _create_exporter(cls, cleanup=True):
-        body = {
-            "name": uuid4(),
-            "repositories": [r.pulp_href for r in cls.export_repos],
-            "path": "/tmp/{}".format(uuid4()),
-        }
-        exporter = cls.exporter_api.create(body)
-        return exporter
-
-    @classmethod
-    def _create_export(cls):
-        export_response = cls.exports_api.create(cls.exporter.pulp_href, {})
-        monitor_task(export_response.task)
-        task = cls.client.get(export_response.task)
-        resources = task["created_resources"]
-        export_href = resources[0]
-        export = cls.exports_api.read(export_href)
-        return export
-
-    @classmethod
-    def _create_chunked_export(cls):
-        export_response = cls.exports_api.create(cls.exporter.pulp_href, {"chunk_size": "5KB"})
-        monitor_task(export_response.task)
-        task = cls.client.get(export_response.task)
-        resources = task["created_resources"]
-        export_href = resources[0]
-        export = cls.exports_api.read(export_href)
-        return export
-
-    @classmethod
-    def _delete_exporter(cls, delete_file=True):
-        """
-        Utility routine to delete an exporter.
+from pulp_rpm.tests.functional.constants import RPM_KICKSTART_FIXTURE_URL, RPM_UNSIGNED_FIXTURE_URL
 
-        Also removes the export-directory and all its contents.
-        """
-        if delete_file:
-            cli_client = cli.Client(cls.cfg)
-            cmd = ("rm", "-rf", cls.exporter.path)
-            cli_client.run(cmd, sudo=True)
-        cls.exporter_api.delete(cls.exporter.pulp_href)
+from pulpcore.client.pulp_rpm import RpmRepositorySyncURL
 
-    @classmethod
-    def tearDownClass(cls):
-        """Clean up."""
-        for remote in cls.remotes:
-            cls.remote_api.delete(remote.pulp_href)
+ExportFileInfo = namedtuple("ExportFileInfo", "output_file_info")
 
-        for repo in cls.export_repos:
-            cls.repo_api.delete(repo.pulp_href)
 
-        for repo in cls.import_repos:
-            cls.repo_api.delete(repo.pulp_href)
+@pytest.fixture
+def create_chunked_export(exporters_pulp_exports_api_client, create_exporter, monitor_task):
+    def _create_chunked_export(import_repos, export_repos, url=RPM_UNSIGNED_FIXTURE_URL):
+        exporter = create_exporter(import_repos, export_repos, url=url)
+        export_response = exporters_pulp_exports_api_client.create(
+            exporter.pulp_href, {"chunk_size": "5KB"}
```

Comment:
Should chunk size be configurable?  That's a question, I'm not as familiar with it so I have no strong opinions.

Reply:
It can be configurable. But I do not see any added value. Do you suggest using `@pytest.mark.parametrize("chunk_size", ["5KB", "5MB", "0KB"])`?